templates/gmcq.hbs

```diff
@@ -7,6 +7,9 @@
             <label aria-hidden="true" id="{{../_id}}-{{@index}}-aria" for="{{../_id}}-{{@index}}" class="{{#unless ../_isEnabled}}disabled {{/unless}} component-item-color component-item-text-color component-item-border {{#if _isSelected}}selected{{/if}} a11y-ignore" tabindex="-1">
 
                 <img src="{{_graphic.src}}" data-large="{{_graphic.large}}" data-small="{{_graphic.small}}"/>
+                {{#if _graphic.attribution}}
+                    <div class="gmcq-item-graphic-attribution">{{{_graphic.attribution}}}</div>
```

Comment:
Happy to do this, I though it best this way, but @oliverfoster thought the other way. See adaptlearning/adapt-contrib-accordion/pull/81